lib/ssh-keys.nix

```diff
@@ -10,6 +10,8 @@
   #
   devOps = {
     craige         = [ "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDtjE0YstRzlh+Zhlj03th9DYOkMqJ5xHUcderBq151K craige@mcwhirter.io"];
+    craige         = [ "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIMZaQ791pDqj6ysu/XurX8VdUUD+L20Y4AqsoBZgtzOn craige@clar"];
+    craige         = [ "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAID7qAXTCAnqq+3ks4L8/2f4J8RxmrFaMOCA7m9ImbW2m craige@sealgair"];
```

Comment:
you need to put all your keys into a single list. eg. https://github.com/input-output-hk/iohk-ops/blob/c236c255f612742732f0a9acd1ca823291438434/lib/ssh-keys.nix#L20